src/ex4/dist/clients/item_client.js

```diff
@@ -20,9 +20,18 @@ class ItemClient {
         }
     }
 
-    async deleteTodo(value) {
+    async updateStatus(id, status) {
         try {
-            const { data } = await axios.delete(`${API_BASE}/${value}`);
+            const { data } = await axios.put(`${API_BASE}/${id}`, { status });
```

Comment:
use a general update client method so you could re-use it for different/future cases